viewer/vue-client/src/components/inspector/item-local.vue

```diff
@@ -563,7 +563,7 @@ export default {
       :all-values-from="allValuesFrom"
       :some-values-from="someValuesFrom"
       :all-search-types="allSearchTypes"
-      :entity-type="entityType" 
+      :entity-type="item['@type']"
```

Comment:
Note that we set `:entity-type="item['@type']"` in the same way on lines 536 and 551 above.